server/websocket.js

```
@@ -0,0 +1,69 @@
+const WebSocket = require("ws");
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)